portal-process/pom.xml

```diff
@@ -141,7 +141,6 @@
     
 	<build>
 		<plugins>
-
```

Comment:
This file is not supposed to be edited? 
Can we revert it, if it's checked-in by mistake

Reply:
Thanks Rahul! I got rid of them